scvi/data/_manager.py

```diff
@@ -25,6 +26,23 @@
 from .fields import AnnDataField
 
 
+@dataclass
+class AnnDataManagerValidationCheck:
+    """
+    Validation checks for AnnorMudata scvi-tools compat.
+
+    Parameters
+    ----------
+    check_if_view
+        If True, checks if AnnData is a view.
+    check_fully_paired_mudata
+        If True, checks if MuData is fully paired across mods.
+    """
+
+    check_if_view: bool = True
+    check_fully_paired_mudata: bool = True
+
+
```

Comment:
@justjhong what do you think?

Reply:
seems fine, the user will never interact with this right?

Reply:
correct